src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -4573,6 +4573,122 @@ public void WriteIntegerValue(bool formatted, bool skipValidation, int value)
             }
         }
 
+        [Theory]
+        //[InlineData(true, true, (float)0)]
```

Comment:
I would remove these.

Reply:
Done.

I tried `dotnet msbuild /t:rebuildandtest /p:TargetGroup=netfx` but gives me following error,

```
C:\Program Files\dotnet\sdk\3.0.100-preview7-012821\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error NETSDK1005: Assets file 'D:\WinCPP\corefx\artifacts\obj\System.Text.Json.Tests\project.assets.json' doesn't have a target for '.NETFramework,Version=v4.7.2'. Ensure that restore has run and that you have included 'net472' in the TargetFrameworks for your project. [D:\WinCPP\corefx\src\System.Text.Json\tests\System.Text.Json.Tests.csproj]
```
So I tried building all test cases at `CoreFx` level by issuing `build -build -buildtests -framework netfx` and it gives same error. Not sure... although I have submitted changes to this PR.

Reply:
I have hit this before a couple of times, particularly when flipping between netcoreapp and netfx test runs. If you look at the project.assets.json file, probably the net472 target is missing. Try re-building from root and see if that fixes the assets file. Re-run:
`build.cmd -framework netfx`

And then run the test again. 
`dotnet msbuild src/System.Text.Json/tests/System.Text.Json.Tests.csproj /t:RebuildAndTest /p:TargetGroup=netfx`

You should see ".NETFramework,Version=v4.7.2" within the list of targets inside `...\corefx\artifacts\obj\System.Text.Json.Tests\project.assets.json`

Reply:
Here's an easier fix that would be good to try, thanks to @safern. Restore the test project before running it:
```text
dotnet msbuild /t:restore;buildandtest /p:TargetGroup=netfx
```

Reply:
Yup I had initially built entire source from root. However @safern 's method worked. Just that I had to do a `build -clean` at root and then rebuild whole code with `build -framework netfx` followed by running test. Don't know if I took a long path :-) but it surely worked. Thanks!!